core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1226,6 +1228,29 @@ public void cancel(@Nullable String message, @Nullable Throwable cause) {
     }
   }
 
+  private static final ClientCall<Object, Object> NOOP_CALL = new ClientCall<Object, Object>() {
```

Comment:
Note that this is copied from `ClientInterceptors.NOOP_CALL` and `DelayedClientCall.NOOP_CALL`. Not refactoring and reusing the code, because this makes minimum change to backport.